main_exercise/client/src/Components/TodoItem/TodoItemConnector.js

```diff
@@ -0,0 +1,15 @@
+import {connect} from 'react-redux'
+import { bindActionCreators } from 'redux'
+import {deleteTodoAction, updateStatusAction, updateUrgencyAction} from "../../actions/todoList-actions";
+import TodoItemComponent from "./TodoItemComponent";
+
+const mapDispatchToProps = (dispatch, ownProps) => {
+    return {
```

Comment:
```suggestion
    return bindActionCreators({ updateStatusAction, updateUrgencyAction, deleteTodoAction }, dispatch);
```

Then you just need to change the names of functions in the component itself from updateStatus to updateStatusAction and so on